src/daemon/demo.sh

```diff
@@ -302,30 +302,23 @@ function bootstrap_mgr {
 # SREE #
 ########
 function bootstrap_sree {
-  if [ ! -f sree.tar.gz ]; then
-    if [ -z "$SREE_VERSION" ]; then
-      sree_latest=$(curl -s 'https://api.github.com/repos/leseb/Sree/releases/latest' | grep tarball_url | cut -d '"' -f 4)
-      curl -L "$sree_latest" -o sree.tar.gz
-    else
-      curl -L https://github.com/leseb/Sree/archive/"$SREE_VERSION".tar.gz -o sree.tar.gz
-    fi
-    mkdir sree && tar xzvf sree.tar.gz -C sree --strip-components 1
-
+  if [ ! -f /Sree-0.1/sree_done ]; then
     ACCESS_KEY=$(awk '/Access key/ {print $3}' /ceph-demo-user)
     SECRET_KEY=$(awk '/Secret key/ {print $3}' /ceph-demo-user)
 
-    pushd sree
+    pushd Sree-0.1
```

Comment:
I'm not version found of the versionned tree. Why not extracting the tarball in a fixed name dir as you did before ? This way, we have fewer code to modify when we update Sree.